docs/earthpy-data-subsets.rst

```diff
@@ -0,0 +1,83 @@
+=====================
+EarthPy Data Subsets
+=====================
+
+EarthPy comes with the ability to download example datasets using
+``earthpy.data.get_data()``. These datasets are used in example documentation
+on how to use EarthPy, as well as in the EarthPy related lessons on
+earthdatascience.org. There are many keys that can be used to get data
+this way, all of which can be found below.
+
+Data Keys
+=========
+
+co-flood-extras
+---------------
+
+Contains two additional csv files for the Colorado Flood exercise on earthdatascience.org.
```

Comment:
Capitalize CSV.  Throughout this document, I would revise the descriptions to make them complete sentences and remove any mention of the earthdatascience.org, lessons, or exercises. Because the website has already been mentioned in the top description, it really doesn't need to be mentioned in the data key description. 

Focus more on the dataset and what it provides, similar to what you started for colorado-flood and spatial-vector-lidar. You can find a lot of this information on the Figshare site/description but sometimes opening the data can be useful. So for this example: 

This dataset contains two supplemental CSV files for the 2013 Colorado Flood data. The files include precipitation for August 21, 2013 to October 11, 2013 in Boulder, Colorado as well as an example of temperature data that includes both null and nodata values for use in a data cleaning process.